### PR TITLE
Fast ladder climb

### PR DESCRIPTION
**Pull request by NewYearsGift**

### CODE REVIEW

**Comment by dapaster** [comment]
any preview?

**Comment by Cyk-Fad** [comment]
maybe it makes legitbot better

**Comment by r3muxd** [comment]
it's not a legit feature, your viewangle goes straight down or straight up when you get on a ladder

File: Osiris/Hacks/Misc.cpp [hunk 1]
```diff
@@ -34,6 +34,60 @@
...
+                }
+            }
+            else if (cmd->viewangles.y < -0 && cmd->viewangles.y > -45 || cmd->viewangles.y < 45 && cmd->viewangles.y > 0)
+            {
+                if (cmd->viewangles.y != 89.20f || !(cmd->buttons & UserCmd::IN_MOVERIGHT))
+                {
+                    cmd->viewangles.y = 89.20f;
+                    cmd->buttons |= UserCmd::IN_MOVERIGHT;
+                }
+            }
```
Comment (danielkrupinski) [inline Osiris/Hacks/Misc.cpp:86]: This could be condensed to < 10 lines of code.

**Review by danielkrupinski — CHANGES_REQUESTED** [review]

**Comment by 3Ld4D** [comment]
this should be done with sidemove and forward move, you can take a look at proper autostrafer which doesn't change viewangles

**Comment by newcommerdontblame** [comment]
Dude its super easy to do without cheat and this  viewangle thing will cause ow ban for sure

**Pull request closed without merging**